### PR TITLE
Sort imports

### LINKED ISSUE #4: fix PEP8 violations

### LINKED ISSUE #48: Sort imports per django's guidelines
Sort all imports per [django's guidelines](https://docs.djangoproject.com/en/1.9/internals/contributing/writing-code/coding-style/#imports) and add  some words to the contributor guidelines reminding folks to stick to those guidelines. 

(I'm breaking #4 into three separate issues.)

### PR DESCRIPTION
**Pull request by kevindaum**
This a second try at https://github.com/SexualHealthInnovations/callisto-core/pull/17. 

This is a work-in-progress. It will close #48. 

I have started by sorting imports, using [django's rules](https://github.com/django/django/blob/master/setup.cfg#L10-L17). Next step is to fix the rest of the pep8 violations. I may also fix issues spotted by pyflakes as I go.

### CODE REVIEW

**Comment by kelseyq** [comment]
Kevin you are awesome! Thanks for rolling this boulder back up the hill!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -4,6 +4,9 @@ Thank you so much for considering contributing to callisto-core. We couldn't do
 
 These guidelines are a work in progress and will be updated as we refine our contributing process. If you have any questions, don't hesitate to ask.
 
+We are adhering to [Django's coding style
```
Comment (kelseyq) [inline CONTRIBUTING.md]: 👍

**Comment by kevindaum** [comment]
@kelseyq In the process of resolving conflicts with master I ran the tests and not all of them passed. I got thirteen `FileNotFoundError: [Errno 2] No such file or directory: 'gpg'` errors and this failure: 

```
FAIL: test_done_saves_anonymised_qs (tests.callistocore.test_views.RecordFormIntegratedTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/kevin/envs/callisto/lib/python3.5/site-packages/mock/mock.py", line 1305, in patched
    return func(*args, **keywargs)
  File "/Users/kevin/projects/other/callisto-core/tests/callistocore/test_views.py", line 196, in test_done_saves_anonymised_qs
    mock_eval_row.save.assert_any_call()
  File "/Users/kevin/envs/callisto/lib/python3.5/site-packages/mock/mock.py", line 999, in assert_any_call
    ), cause)
  File "<string>", line 2, in raise_from
AssertionError: mock() call not found
```

I don't think any of those issues are the result of my merge so I'm just going to move on.

**Comment by kevindaum** [comment]
I'm assuming this from https://travis-ci.org/SexualHealthInnovations/callisto-core/jobs/140262027 is why the Travis builds are failing:

```
$ pyflakes setup.py runtests.py tests callisto
tests/callistocore/test_delivery.py:6: redefinition of unused 'patch' from line 3
tests/callistocore/test_delivery.py:6: redefinition of unused 'call' from line 3
tests/callistocore/test_views.py:4: redefinition of unused 'patch' from line 2
tests/callistocore/test_views.py:4: redefinition of unused 'Mock' from line 2
callisto/delivery/forms.py:10: redefinition of unused 'urlsplit' from line 1
callisto/delivery/forms.py:10: redefinition of unused 'parse_qs' from line 1
```

Looks like we're using `mock` instead of `unittest.mock` and `six.moves.urllib.parse` instead of `urllib.parse` now... just correct me if I'm wrong 😄.

**Comment by kevindaum** [comment]
@kelseyq I'm starting to think that sorting imports, fixing pep8 violations and fixing any issues brought up by pyflakes should be three separate pull requests, just because of how many files are involved (and therefore how much of a pain it is to keep this branch up to date with changes in master). Would that be okay with you? I could create new issues for imports and pyflakes so that each PR has an issue.

**Comment by kevindaum** [comment]
FYI, I opened an issue with isort to make the sorting per django's coding style guidelines automatic: https://github.com/timothycrosley/isort/issues/443.

**Comment by kelseyq** [comment]
Separate PRs make a lot of sense to me for this big of a chunk of changes--it'll also help me responsibly review them if they're in smaller chunks. Great idea! Can you go ahead and make separate tickets so that others know you're working on them?

**Comment by kevindaum** [comment]
@kelseyq this is ready for review.

**Comment by kelseyq** [comment]
This looks great, @kevindaum! I'd like to add instructions to run isort to CONTRIBUTING.md, but I think it would make sense to do that after #38 is merged in. I went ahead and created #57 to track that.

I'd like to see code that fails isort to fail the build. There's discussion in this bug of using the `--check-only` option to do this: https://bugzilla.mozilla.org/show_bug.cgi?id=1192957

I'm fine with creating that as a separate ticket if you'd like to merge this in first. Let me know.

**Comment by kevindaum** [comment]
@kelseyq I prefer that you merge this and create a separate ticket for adding isort to Travis. I'm new to Travis (and to isort), so it might take me a minute to figure out.

**Pull request merged**